solidity/apps/test/governance/governanceRouter.test.ts

```diff
@@ -40,10 +41,13 @@ describe('GovernanceRouter', async () => {
         address: governor.address,
       },
     };
+    await abacus.deploy(domains, governor);
```

Comment:
Is because of the the funky side effects of the tests?

Reply:
yup